tasks.py

```diff
@@ -103,7 +103,9 @@ def minimum(c):
 def lint(c):
     check_dependencies(c)
     c.run('flake8 ctgan')
+    c.run('pydocstyle ctgan')
     c.run('flake8 tests --ignore=D,SFS2')
```

Comment:
Change this to `flake8 tests` without ignoring any error code.

Reply:
@pvk-developer If we remove the error code D101, than lint complains about a docstring for the test classes, like this one:

https://github.com/sdv-dev/CTGAN/blob/5c0f5bfa05f53047c43cc17f8522fa4083581e55/tests/unit/synthesizer/test_ctgan.py#L12

Most of our libraries don't have docstrings for these cases at all. Does that mean we should change that in all libraries, or should we just ignore D101 from the `flake8 test` command?

Reply:
🤔 I actually added a small description on them such as: `"""Test class for Discriminator"""` .

Reply:
After discussing this privately, we concluded that writing quick docstrings that don't really clarify much is not a good practice, so instead we will simply ignore the D101 error in the `flake8 test` command.